### PR TITLE
Move window/UMD bridge in `netlify-cms` instead of `netlify-cms-core`

### PR DESCRIPTION
**Pull request by LoicMahieu**
**Summary**

`netlify-cms` package is responsible to make NetlifyCMS easy to use. It is bundled with the UMD wrapper and is used as script tag. It imports all widgets and backends...
It makes sense that `netlify-cms` should handle whole aspects about this "window bridge"... and keep `netlify-cms-core` as a clean ESM module.

It will make `netlify-cms-core` easy to use as a standalone ESM module:
```
import { init } from "netlify-cms-core"
init({})
```

Without this PR, a `window.CMS_MANUAL_INIT` should be set before the `import` is done.

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 32608330205242b9d6a560febfd356662dea1bb5

https://deploy-preview-1865--netlify-cms-www.netlify.com

**Comment by talves** [comment]
@LoicMahieu I had this change on a branch on my system and forgot about it before I tested. Thanks for doing this and reminding me to test this out.

@erquhart I think this **might** be a viable move, since it really does not change anything in the api or exports.

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 06ffbc1bc3b619d056b7881a122b7cd2aab556d7

https://deploy-preview-1865--cms-demo.netlify.com

**Comment by LoicMahieu** [comment]
After that I intent to submit [that](https://github.com/LoicMahieu/netlify-cms/commit/2c543f2ea3c4537d7cfb3dee93b74269d5a62833), would not that be right here ?

**Comment by talves** [comment]
I will let @erquhart make that call. 

I think it makes sense to keep it separate to test apart, but release them both at the same time. Might not make a difference though.

File: packages/netlify-cms/src/index.js [hunk 1]
```diff
@@ -1,7 +1,28 @@
+import createReactClass from 'create-react-class';
```
Comment (erquhart) [inline packages/netlify-cms/src/index.js]: This dependency needs to be installed in the `netlify-cms` package and removed from `netlify-cms-core`.

**Comment by talves** [comment]
@LoicMahieu Thanks for working on this. It is going to take Netlify CMS a lot closer to being able to be used as a component.

**Pull request merged**